pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "opower"
-version = "0.0.23"
+version = "0.0.24"
```

Comment:
Revert so that I can manage the release myself. There are a few pending PRs, some I just merged, some almost to be merged. It's better for me to bump this when I'm ready.